owa/app/js/components/overview-visits/table/index.tsx

```diff
@@ -93,7 +93,7 @@ export default class OverviewVisitTable extends React.PureComponent<PropsWithInt
     const loadingText = <FormattedMessage id="visits.overviewLoadingLabel" />;
     const pageText = <FormattedMessage id="visits.overviewPageLabel" />;
     const ofText = <FormattedMessage id="visits.overviewOfLabel" />;
-    const rowsText = <FormattedMessage id="visits.overviewResultsLabel" />;
+    const rowsText = this.props.intl.formatMessage({id: "visits.overviewResultsLabel"});
```

Comment:
why only this one has been changed to this.props.intl.formatMessage? I would change all these consts (if possible) to be consistent